### ISSUE TITLE
SaveSrc not working

### ISSUE THREAD

**Issue by hollyhook** [tixl3d/tixl#214]
SaveSrc button in the output window not having any effect.

I was expecting a screenshot being saved in .t3\screenshots. 

dev branch, on revision17ff65e

**Comment by hollyhook** [comment]
Issue somewhat confirmed by sense, see https://discord.com/channels/823853172619083816/986939763992981524/1038347441268862976

**Comment by hollyhook** [comment]
See https://discord.com/channels/823853172619083816/986939763992981524/1038383477705420851

**Comment by pixtur** [comment]
Yes. Sadly it looks like the first couple of clicks, the GPU hasn't computed a valid result. This needs to be fixed/

**Comment by pixtur** [comment]
Fixed with v3.9
